docs/en/latest/plugins/opentelemetry.md

```diff
@@ -26,160 +27,186 @@ description: This document contains information about the Apache opentelemetry P
+    resource:                         # Additional resource to append to the trace.
+      service.name: APISIX            # Set the Service name for OpenTelemetry traces.
     collector:
-      address: 192.168.8.211:4318
-      request_timeout: 3
-      request_headers:
-        foo: bar
-    batch_span_processor:
-      drop_on_queue_full: false
-      max_queue_size: 6
-      batch_timeout: 2
-      inactive_timeout: 1
-      max_export_batch_size: 2
+      address: 127.0.0.1:4318       # Set the address of the OpenTelemetry collector to send traces to.
+      request_timeout: 3            # Set the timeout for requests to the OpenTelemetry collector in seconds.
+      request_headers:              # Set the headers to include in requests to the OpenTelemetry collector.
+        Authorization: token        # Set the authorization header to include an access token.
+    batch_span_processor:           # Trace span processor.
+      drop_on_queue_full: false     # Drop spans when the export queue is full.
+      max_queue_size: 1024          # Set the maximum size of the span export queue.
+      batch_timeout: 2              # Set the timeout for span batches to wait in the export queue before
+                                    # being sent.
+      inactive_timeout: 1           # Set the timeout for spans to wait in the export queue before being sent,
+                                    # if the queue is not full.
+      max_export_batch_size: 16     # Set the maximum number of spans to include in each batch sent to the OpenTelemetry collector.
+    set_ngx_var: false              # Export opentelemetry variables to nginx variables.
 ```
 
-## Variables
+Reload APISIX for changes to take effect.
 
-The following nginx variables are set by OpenTelemetry:
+## Attributes
 
-- `opentelemetry_context_traceparent` -  [W3C trace context](https://www.w3.org/TR/trace-context/#trace-context-http-headers-format), e.g.: `00-0af7651916cd43dd8448eb211c80319c-b9c7c989f97918e1-01`
-- `opentelemetry_trace_id` - Trace Id of the current span
-- `opentelemetry_span_id` -  Span Id of the current span
+| Name                                  | Type          | Required | Default      | Valid Values | Description |
+|---------------------------------------|---------------|----------|--------------|--------------|-------------|
+| sampler                               | object        | False    | -            | -            | Sampling configuration. |
+| sampler.name                          | string        | False    | `always_off` | `always_on`, `always_off`, `trace_id_ratio`, or `parent_base` | Sampling strategy.<br>To always sample, use `always_on`.<br>To never sample, use `always_off`.<br>To randomly sample based on a given ratio, use `trace_id_ratio`.<br>To use the sampling decision of the span's parent, use `parent_base`. If there is no parent, use the root sampler. |
```

Comment:
```suggestion
| sampler.name                          | string        | False    | `always_off` | ["always_on", "always_off", "trace_id_ratio", "parent_base"]  | Sampling strategy.<br>To always sample, use `always_on`.<br>To never sample, use `always_off`.<br>To randomly sample based on a given ratio, use `trace_id_ratio`.<br>To use the sampling decision of the span's parent, use `parent_base`. If there is no parent, use the root sampler. |
```

for valid values I guess use the apache/apisix format to avoid confusions, since the valid values below and elsewhere are also using the `[]` annotations?

<img width="88" alt="image" src="https://github.com/user-attachments/assets/58c2ad76-65ac-491b-bdc9-443fa9f9dcba" />